doc/NEWS

```diff
@@ -117,6 +117,11 @@ Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
   used before compression.  Before this we had false negatives for those.
   Also updated LZMA code from upstream, to versioj 19.00.
 
+- Upgraded Unicode stuff from version 11.0.0 to 13.0.0 (this affects the
+  external modes Dumb16, Dumb32, Repeats16 and Repeats32.  It also affects
+  the eastern-egg option "--subsets=full-unicode" which BTW supersedes the
+  external modes with far better performance.
```

Comment:
Missed closing brace for "(this affects ...".

For the eastern-egg option, I think we should mention it in comments near the external mode definitions. I think we should similarly mention the "--subsets" option in general near the Subsets external mode definitions.

Reply:
Thanks, I'll fix that soon.

Reply:
Also, s/eastern-egg/Easter egg/ and you need to add your nickname and year like we've been doing on other NEWS entries.